### PR TITLE
Add responsive sizing of VCAuthn login page for mobile devices

### LINKED ISSUE #420: Responsive sizing for VCAuthn login page on mobile devices
Placeholder, to expand here. The page seems to just shrink on mobile devices due to fixed sizing. Things get quite small

Ex:
image

### PR DESCRIPTION
**Pull request by Gavinok**
This PR resolves #420 using media queries as well as bootstrap.

Here is the new desktop view

To ensure uniform rendering of the boarder it no longer uses a dedicated svg and instead uses CSS boarders

On mobile devices the card style is removed and instead the entire screen is used to make the most of the smaller displays.

### CODE REVIEW

**Comment by Gavinok** [comment]
With the removal of media queries the css is simplified. I have also migrated the button css to make use of bootstrap. Here is the updated look on desktop



and on mobile

**Comment by esune** [comment]
@Gavinok can you resolve the conflicts - likely with code from the last merged PR?

**Comment by Gavinok** [comment]
@esune Ready for review

**Comment by Gavinok** [comment]
@esune resolved the conflict

**Review by esune — DISMISSED** [review]
LGTM based on screenshots and what demonstrated during live review.

**Review by loneil — CHANGES_REQUESTED** [review]
Holding this as deep link is no longer delivering request message (working on main fine)

**Comment by loneil** [comment]
@Gavinok see comment above, not working with form action, needs to be an href link I believe.

Probably: https://stackoverflow.com/questions/45542773/how-to-style-an-anchor-as-a-button-in-bootstrap-4

File: oidc-controller/api/templates/verified_credentials.html [hunk 1]
```diff
@@ -386,7 +383,9 @@ <h1 class="mobile-device">Continue with:</h1>
             rows="3"
             disabed
             value="{{url_to_message}}"
-          >{{url_to_message}}</textarea>
+          >
+{{url_to_message}}</textarea
+          >
```
Comment (loneil) [inline oidc-controller/api/templates/verified_credentials.html:388]: Formatting

**Comment by loneil** [comment]
Can we do some additional tweaks to the widths of the "header-desc" (and possibly other) containers?

They're thinner in which is crowding texts.

Existing:
image

Current:

Initial text (also could use a top margin here when opening the "other device" section)
image

Declined
image
(Probably same with approved and expired states)

Scanned mask over the qr code is thinner
image

**Comment by Gavinok** [comment]
> @Gavinok see comment above, not working with form action, needs to be an href link I believe.
> 
> Probably: https://stackoverflow.com/questions/45542773/how-to-style-an-anchor-as-a-button-in-bootstrap-4

Dang that was what made is able to remove all that custom button CSS :(

**Comment by esune** [comment]
I'm wondering if we should tackle the UI redesign/tweaks as part of the full move to Bootstrap, rather than potentially do the work twice? https://github.com/bcgov/vc-authn-oidc/issues/431

**Comment by loneil** [comment]
> > @Gavinok see comment above, not working with form action, needs to be an href link I believe.
> > Probably: https://stackoverflow.com/questions/45542773/how-to-style-an-anchor-as-a-button-in-bootstrap-4
> 
> Dang that was what made is able to remove all that custom button CSS :(

No should just need to change `button` to `a` I just did that on my local and it worked.

**Comment by Gavinok** [comment]
> Reference in new issue
> Edit
> Hide
> Delete

It was more to do with the css not indicating the button was disabled but I was able to work around this anyways

**Comment by Gavinok** [comment]
@loneil is it just me or does the qr code expire extremely fast now? Did I make a change that could effect that?

**Comment by Gavinok** [comment]
> Can we do some additional tweaks to the widths of the "header-desc" (and possibly other) containers?
> 
> They're thinner in which is crowding texts.
> 
> Existing: image
> 
> Current:
> 
> Initial text (also could use a top margin here when opening the "other device" section) image
> 
> Declined image (Probably same with approved and expired states)
> 
> Scanned mask over the qr code is thinner image

I have just updated this. There should be a little more space. I figured we should add some line breaks for more consistent text formatting while I was at it.

**Review by esune — APPROVED** [review]

**Review by loneil — APPROVED** [review]
Works for me, should be good to deploy for more device testing in dev. Can refine further with bootstrap framework.

**Pull request merged**